src/lib/optimizer/strategy/index_scan_rule.cpp

```diff
@@ -71,6 +71,8 @@ bool IndexScanRule::_is_index_scan_applicable(const IndexStatistics& index_stati
 
   if (index_statistics.column_ids[0] != operator_predicate.column_id) return false;
 
+  return true;
```

Comment:
What about the rest of the code in line 76-83? Am I wrong or do you always return here?

Reply:
We temporarily always use the indexScan when applicable, and not only when the selectivity rating is good enough, to test the performance. I added a TODO